backend/handler/db.js

```diff
@@ -27,21 +27,61 @@
 		autoReconnect: true
 	};
 
+	function getGridFSBucket(database, collection, chunksize = null) {
+		return Handler.getDB(database).then(dbConn => {
```

Comment:
I know you just moved functions around, but while we're here can we rewrite this to async await?